testing/src/main/java/io/stargate/it/http/graphql/graphqlfirst/DeleteCustomConditionsTest.java

```diff
@@ -97,6 +94,11 @@ public static void setup(
             + "    vs: [Int] @cql_if(field: \"v\", predicate: IN)\n"
             + "    ): Boolean\n"
             + "    @cql_delete(targetEntity: \"Foo\")\n"
+            + "  deleteFooWithResponsePayload(\n"
```

Comment:
do we have a corresponding test for an update?

Reply:
Yes, see `UpdateCustomConditionsTest`.